lesson-3/Berman-lesson3/index.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Home page</title>
+  </head>
+  <body>
+
+
+    <h1>Home!</h1>
+
+    <a href="About/">Link to about page</a><br/>

```

Comment:
It won't make a huge difference here, but make sure to always write your links exactly how the folders are named. In this case you named your folders with lowercase letters, but wrote them capitalized. This should be `about/`

Reply:
On my computer the folders show capital letters. Does this change when i upload it here?

Reply:
Odd, you're right. Not sure where I saw them lowercase haha